automating_bulk_processor.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+python -m toolbox.bulk_processing.refusal_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "refusal_processor has failed"
```

Comment:
these echo'ed errors will be harder to grep out of the logs won't they?

Reply:
I've created a metric in the terraform that picks up the log lines and we can alert on that